docs/getting-started/pingfederate.md

```diff
@@ -125,7 +128,7 @@ terraform {
   required_version = ">=1.4"
   required_providers {
     pingfederate = {
-      version = ">= 0.1.0, < 1.0.0"
+      version = "~> 1.0"
```

Comment:
This should probably be `version = ">= 1.0, < 2.0"` so anyone copying/pasting will get the latest updates, or we're committing to update this document on each minor version release